client/src/components/HistoryViewer/HistoryViewer.js

```diff
@@ -136,25 +136,42 @@ class HistoryViewer extends Component {
       recordClass,
       schemaUrl,
       VersionDetailComponent,
+      compareMode,
+      compareFrom,
+      compareTo,
```

Comment:
Have we switched this to `compare` as an object slash bool now?

Reply:
Not at this level, I only made the change for the part that was not connected to the store. Although that said, because it's a map function, it too should be easy to alter.

Reply:
@ScopeyNZ has told use he is taking care of this in a separate PR :)